src/wazuh_db/wdb.h

```diff
@@ -1061,6 +1062,18 @@ int wdb_parse_global_update_agent_data(wdb_t * wdb, char * input, char * output)
  */
 int wdb_parse_global_get_agent_labels(wdb_t * wdb, char * input, char * output);
 
+/**
+ * @brief Function to check if there is at least one agent that needs to synchronize the group information.
+ *        If not, it looks for mismatch between the calculated hash and the received one.
+ *
+ * @param wdb The global struct database.
+ * @param input String with 'agent_id'.
```

Comment:
Typo: input will hold a hash to be compared, not agent_id

Reply:
Changed!